README.md

```diff
@@ -28,14 +28,17 @@ them with your package manager.
 Else you can install them manually
 
 ### Meson
+You need to install python 3 and pip. On debian and ubuntu, they are provided by the package python3-pip
 
 ```
-pip install meson --user # Install Meson
+pip3 install meson --user # Install Meson
+cp ~/.local/bin/meson <a_directory_in_your_path>
```

Comment:
Does `~/.local/bin` not already in the `$PATH` ?
It's seems to be the case on Fedora and Ubuntu. (And that's way `pip` install binaries in this directory)